openproblems/data/pancreas.py

```diff
@@ -53,7 +53,7 @@ def load_pancreas(test=False, integer_only=False):
         adata = sc.read(filepath)
 
     # NOTE: X contains counts that are normalized with scran
-    adata.layers["log_scran"] = adata.X
+    adata.layers["log_scran_pooling"] = adata.X
```

Comment:
I'd actually call this `"log_normalized"` since it's not technically all scran

Reply:
Yes, I think so -- we don't want the global dataloader to inadvertently give people data they think is scran normalized and is actually something else